client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -40,8 +40,6 @@ class WaitingTwoFactorNotificationApproval extends Component {
 
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.pushSuccess && nextProps.pushSuccess ) {
-			const { translate } = this.props;
-			this.props.successNotice( translate( 'Logging In…' ) );
```

Comment:
We actually aren't sure if we want this or not (https://github.com/Automattic/wp-calypso/issues/14155) but it's fine to remove it for now.